### DISCUSSION TITLE
Geliştirilmeye devam ediyor mu?

### DISCUSSION THREAD

**Discussion by bisimit** [realsuayip/django-sozluk#44] [Q&A] [answered]
Merhaba

Emekleriniz için teşekkür ederiz. Projeyi geliştirmeye devam edecek misiniz?

İyi çalışmalar

**Comment by realsuayip** [comment, marked as answer]
Merhaba, proje şu an tamamen feature-complete. Herhangi bir bug bulunması durumunda en kısa sürede çözülür. Yani şu an proje maintain ediliyor, uzun bir süre de maintain edeceğim.

Yeni özellik eklemeyi düşünmüyorum, olağanüstü bir talep olmadığı sürece (örneğin ~10 kadar kişi yeni bir özellik üzerine talepte bulunursa).
